pkg/backend/filestate/state.go

```diff
@@ -553,3 +554,24 @@ func (b *localBackend) addToHistory(ref *localBackendReference, update backend.U
 	checkpointFile := fmt.Sprintf("%s.checkpoint.%s", pathPrefix, ext)
 	return b.bucket.Copy(context.TODO(), checkpointFile, b.stackPath(ref), nil)
 }
+
+// isStateEmpty reports whether the state inside the given bucket is empty.
+func isStateEmpty(ctx context.Context, b Bucket) (bool, error) {
```

Comment:
Does this refer to the filestate backend not having any state files in it? I find it confusing between term "state" being used to refer to the filestate backend and pulumi "state" (files).

Reply:
Oops, this means that the Pulumi part of the bucket is empty. The '.pulumi' directory.
Let me rename it to something better.